clang/test/CodeGen/PowerPC/inline-asm-unsupported-constraint-error.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -emit-llvm -triple powerpc64le-linux-gnu -verify %s
```

Comment:
I think one RUN line (powerpc64) is sufficient. Duplicating this for ELF/XCOFF isn't necessary.

`inline-asm-constraints-error.c` might be a better test name. You can reuse the test for other errors.